crates/networking/rpc/eth/block.rs

```diff
@@ -311,14 +310,9 @@ impl RpcHandler for GetBlobBaseFee {
             Some(header) => header,
             _ => return Err(RpcErr::Internal("Could not get block header".to_owned())),
         };
-        let parent_header = match find_parent_header(&header, &context.storage) {
-            Ok(option_header) => option_header,
-            Err(error) => return Err(RpcErr::Internal(error.to_string())),
-        };
-
         let config = context.storage.get_chain_config();
         let blob_base_fee = calculate_base_fee_per_blob_gas(
-            parent_header.excess_blob_gas.unwrap_or_default(),
+            header.excess_blob_gas.unwrap_or_default(),
```

Comment:
For reference, the same code in geth is: https://github.com/ryanschneider/go-ethereum/blob/13bde9bb8572898abab200be7bd1616f5650257e/eth/gasprice/feehistory.go#L97-L103